src/taxonomy/taxonomy-card/TaxonomyCard.jsx

```diff
@@ -0,0 +1,141 @@
+      break;
+    }
+  };
+
+  const getSystemBadgeToolTip = () => (
+    <Popover id={`system-defined-tooltip-${id}`}>
+      <Popover.Title as="h5">
+        {intl.formatMessage(messages.systemTaxonomyPopoverTitle)}
+      </Popover.Title>
+      <Popover.Content>
+        {intl.formatMessage(messages.systemTaxonomyPopoverBody)}
+      </Popover.Content>
+    </Popover>
+  );
+
+  const getHeaderSubtitle = () => {
+    if (systemDefined) {
+      return (
+        <OverlayTrigger
+          key={`system-defined-overlay-${id}`}
+          placement="top"
+          overlay={getSystemBadgeToolTip(id)}
+        >
+          <Badge variant="light">
+            {intl.formatMessage(messages.systemDefinedBadge)}
+          </Badge>
+        </OverlayTrigger>
+      );
+    }
+    if (orgsCountEnabled()) {
+      return (
+        <div className="font-italic">
+          {intl.formatMessage(messages.assignedToOrgsLabel, { orgsCount })}
+        </div>
+      );
+    }
+    return undefined;
+  };
+
+  const getHeaderActions = () => {
+    if (systemDefined) {
+      // We don't show the export menu, because the system-taxonomies
+      // can't be exported. The API returns and error.
+      // The entire menu has been hidden because currently only
+      // the export menu exists.
+      //
+      // TODO When adding more menus, change this logic to hide only the export menu.
+      return undefined;
+    }
+    return (
+      <TaxonomyCardMenu
+        id={id}
+        name={name}
+        onClickMenuItem={onClickMenuItem}
+      />
+    );
+  };
+
+  const renderModals = () => (
+    // eslint-disable-next-line react/jsx-no-useless-fragment
+    <>
+      {isExportModalOpen && (
+        <ExportModal
+          isOpen={isExportModalOpen}
+          onClose={() => setIsExportModalOpen(false)}
+          taxonomyId={id}
+        />
+      )}
+    </>
+  );
+
+  return (
+    <>
+      <Card className={classNames('taxonomy-card', className)} data-testid={`taxonomy-card-${id}`}>
+        <Card.Header
+          title={name}
+          subtitle={getHeaderSubtitle()}
+          actions={getHeaderActions()}
+        />
+        <Card.Body className={classNames('taxonomy-card-body', {
+          'taxonomy-card-body-overflow-m': !systemDefined && !orgsCountEnabled(),
+          'taxonomy-card-body-overflow-sm': systemDefined || orgsCountEnabled(),
+        })}
+        >
+          <Card.Section>
+            {description}
+          </Card.Section>
+        </Card.Body>
+      </Card>
+      {renderModals()}
+    </>
+  );
+};
+
+TaxonomyCard.defaultProps = {
+  className: '',
+};
+
+TaxonomyCard.propTypes = {
+  className: PropTypes.string,
+  original: PropTypes.shape({
+    id: PropTypes.number,
+    name: PropTypes.string,
+    description: PropTypes.string,
+    systemDefined: PropTypes.bool,
+    orgsCount: PropTypes.number,
+  }).isRequired,
+  intl: intlShape.isRequired,
+};
+
+export default injectIntl(TaxonomyCard);
```

Comment:
Instead of injectIntl it's now better to use the intl hook as:

```
const intl = useIntl();
```

Reply:
I replaced all `injectIntl` with `useIntl` on the taxonomy directory here: https://github.com/openedx/frontend-app-course-authoring/pull/645/commits/61864d3187a343edb2eaa56b643e7114cb237d12